src/personnummer.vue

```diff
@@ -166,6 +166,17 @@ export default {
 
       return true
     },
+    normaliseYear(ssn) {
+      // Append centuries on 10 digit long numbers
+      if (ssn.length === 10) {
+        const year = parseInt(ssn.substring(0, 2))
+        const twoDigitsCurrentYear = parseInt(new Date().getFullYear().toString().substr(2,2));
```

Comment:
`String.prototype.substr()` has effectively been deprecated

```suggestion
        const twoDigitsCurrentYear = parseInt(new Date().getFullYear().toString().substring(2, 4));
```